### PR TITLE
Added Java to Language section

### LINKED ISSUE #527: [Feature Request] Adding Java in the Language section
### Is your feature request related to a problem? Please describe.
The Java is very popular coding language used everywhere in the world, so i would like to introduce its features!

### Describe the solution you'd like
I will add the Java to the languages and make the documentations for it

### Describe alternatives you've considered
no alternatives. Java is a very strong coding language

---

**Would you like to work on this feature?**  
- [x] Yes  
- [ ] No  

Under the `gssoc-ext` label

### PR DESCRIPTION
**Pull request by LitZeus**
### Description
The Java programming language has been added to the `Languages` section
Fixes #527 

image

<br />

image

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/languages/java/java-2.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+Relational operators compare two values and return a boolean result (`true` or `false`).
+
+| Operator | Description | Example |
+| --- | --- | --- |
+| \== | Equal to | x == y |
+| != | Not equal to | x != y |
+| \> | Greater than | x \> y |
+| \< | Less than | x \< y |
+| \>= | Greater than or equal to | x \>= y |
+| \<= | Less than or equal to | x \<= y |
```
Comment (ajay-dhangar) [inline docs/languages/java/java-2.md]: replace `<` with `&lt;`, and `>` with `&gt;`

File: docs/languages/java/java-2.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+## 3. Logical Operators
+
+Logical operators are used to perform logical operations and combine multiple conditions.
+
+| Operator | Description | Example |
+| --- | --- | --- |
+| && | Logical AND | (x \> 5 && y \< 10) |
+|  |  |  |
+| ! | Logical NOT | !(x \> 5) |
```
Comment (ajay-dhangar) [inline docs/languages/java/java-2.md]: The HTML code you provided is using `<` and `>` to create tags. This is causing a conflict with the JavaScript syntax for JSX in React.js, which also uses `<` and `>` to create tags. To resolve this conflict, you can replace the `<` and `>` characters in the HTML or .md code with their HTML entity equivalents, `&lt;` and `&gt;`.

File: docs/languages/java/java-2.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+| Operator | Description | Example |
+| --- | --- | --- |
+| & | Bitwise AND | x & y |
+|  \|  | Bitwise OR | \| |
+| ^ | Bitwise XOR | x ^ y |
+| ~ | Bitwise NOT | ~x |
+| \>\> | Right shift | x \>\> 2 |
+| \<\< | Left shift | x \<\< 2 |
+
```
Comment (ajay-dhangar) [inline docs/languages/java/java-2.md]: also, replace `>`, and `<`

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by LitZeus** [comment]
@ajay-dhangar I have done the changes, shall I make another PR for this?

I have also made the changes in the codes in the CLI as they were coming in the single line.

**Comment by LitZeus** [comment]
@ajay-dhangar When i pulled the main branch to make changes by having the latest repo version i am getting this error!
Though I have made the changes as per you have told.
Please have a look!

image

**Comment by ajay-dhangar** [comment]
> @ajay-dhangar When i pulled the main branch to make changes by having the latest repo version i am getting this error! Though I have made the changes as per you have told. Please have a look!
> 
> image

If you get this error, you need to run the command `npm i` to install our React Lite YouTube Embed (a new package for video content).

**Review by ajay-dhangar — APPROVED** [review]
Done || I'll fix `.md` syntax error

**Pull request merged**